config/ext_datadog_prodpublick8s.yaml

```diff
@@ -0,0 +1,9 @@
+datadog:
+  clusterName: 'prodpublick8s'
+  kubelet:
+    host:
+      valueFrom:
+        fieldRef:
+          fieldPath: spec.nodeName
+    hostCAPath: /etc/kubernetes/certs/kubeletserver.crt
+    tlsVerify: false # Required as of Agent 7.35 because Kubelet certificates in AKS do not have a Subject Alternative Name (SAN) set.
```

Comment:
To be revised?

Reply:
Did AKS started to use a SAN for the kubelet's generated certificate ?